core/src/main/java/org/jdbi/v3/core/statement/SqlStatements.java

```diff
@@ -143,6 +145,7 @@ public TimingCollector getTimingCollector() {
      * @param timingCollector the new timing collector
      * @return this
      */
+    @Deprecated
     public SqlStatements setTimingCollector(TimingCollector timingCollector) {
         this.timingCollector = timingCollector == null ? TimingCollector.NOP_TIMING_COLLECTOR : timingCollector;
```

Comment:
We could get rid of the `timingCollector` field altogether if we wrapped the collector passed into this method in a `SqlLogger` adapter, and passed it to `setSqlLogger()`.

Reply:
Hmm, not a fan of this actually, it's screwing up the test logic. Can we put this back?